data/macvtap/002-macvtap-daemonset.yaml

```diff
@@ -0,0 +1,63 @@
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: macvtap-dp-config
+  namespace: {{ .Namespace }}
+data:
+  DP_MACVTAP_CONF: "[]"
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: macvtap-cni
+  namespace: {{ .Namespace }}
+spec:
+  selector:
+    matchLabels:
+      name: macvtap-cni
+  template:
+    metadata:
+      labels:
+        name: macvtap-cni
+    spec:
+      hostNetwork: true
+      hostPID: true
+      containers:
+      - name: macvtap-cni
+        command: [ "/macvtap-deviceplugin", "-v", "3", "-logtostderr"]
+        envFrom:
+          - configMapRef:
+              name: macvtap-dp-config
+        image: {{ .MacvtapImage }}
+        imagePullPolicy: {{ .ImagePullPolicy }}
+        resources:
+          requests:
+            cpu: "60m"
+            memory: "30Mi"
+          limits:
+            cpu: "60m"
+            memory: "30Mi"
+        securityContext:
+          privileged: true
```

Comment:
Is this needed? I forgot the scope of the macvtap DP. Is it just passing the character device or is it creating it as well?

Reply:
It creates it. We need to be privileged.